package.json

```diff
@@ -27,14 +27,15 @@
     "@typescript-eslint/eslint-plugin": "5.25.0",
     "@typescript-eslint/experimental-utils": "5.25.0",
     "@typescript-eslint/parser": "5.25.0",
+    "@types/pako": "2.0.3",
 
     "rollup": "2.58.0",
     "rollup-plugin-babel": "4.4.0",
     "rollup-plugin-commonjs": "10.1.0",
     "rollup-plugin-node-resolve": "5.2.0",
     "rollup-plugin-terser": "7.0.2",
     "rollup-plugin-typescript2": "0.33.0",
-    "typescript": "4.8.2",
+    "typescript": "4.9.5",
```

Comment:
Updated TypeScript to `4.9.x` so I could use the [satisfies](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) operator. Could also consider updating to latest `5.x`, but that would probably require updating ESLint libraries as well.